src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -473,6 +486,14 @@ private unsafe int HandleEventConnected(ref CONNECTED_DATA data)
             NetEventSource.Info(this, $"{this} Received event CONNECTED {LocalEndPoint} -> {RemoteEndPoint}");
         }
 
+#if DEBUG
+        if (_tlsSecret != null)
```

Comment:
The original ideal was to clear the secrets but it probably does not matter any more since the cleanup was moved the the SafeHandle. I removed it to make less `#if DEBUG` branches.